hoot-test/src/main/cpp/hoot/test/ScriptTestSuite.cpp

```diff
@@ -70,7 +71,10 @@ ScriptTestSuite::ScriptTestSuite(QString dir, bool printDiff, double waitTimeSec
       {
         if (fi.baseName().startsWith(ignorePrefix[j]))
         {
-          LOG_WARN("Disabling: " << fi.filePath());
+          if (!hideDisableTests)
+          {
+            LOG_WARN("Disabling: " << fi.filePath());
+          }
```

Comment:
I'm more of a leave-off-the-curly-braces kind of guy when there is only a single line in an `if` but the `LOG_WARN` macro turns into multiple lines... :dumpster_fire:

Reply:
:thats_horrible: